### ISSUE TITLE
User configurable Color Themes

### ISSUE THREAD

**Issue by migueldeicaza** [gui-cs/Terminal.Gui#29]
Should include other color themes, currently we have the Norton-esque theme, we could include a TurboVision-esque theme.   The support is already in place, it is a matter of allocating new colors in each driver.

**Comment by kostrse** [comment]
Look and feel of Turbo Vision (not only colors) would be really great!

**Comment by tznind** [comment]
> It should be possible for a user to define the colors used for Dialog, Error, Menu, and TopLevel in a configuration file. The drivers would then load these color overrides and use them instead fo the colors built into the source.

Not sure if this helps but up to now I have just used yaml for specifying custom color combinations (but not new color codes):

https://github.com/SMI/IsIdentifiable/blob/main/ii/TerminalGuiTheme.cs
https://github.com/SMI/IsIdentifiable/blob/main/ii/theme.yaml

```csharp
var themeFile = Path.Combine(TestContext.CurrentContext.TestDirectory, "theme.yaml");
var des = new Deserializer();

var theme = des.Deserialize<TerminalGuiTheme>(File.ReadAllText(themeFile));
```

Presumably we don't want a dependency on a specific persistence library though?

**Comment by tig** [comment]
Fixed by #2284
